### DISCUSSION TITLE
Serenity Save Handler Timeout

### DISCUSSION THREAD

**Discussion by ndestcroix** [serenity-is/Serenity#7225] [Help]
I am getting a timeout error from an Update request that was created from serenity code. It gives me a "500 - The request timed out" error but my other requests (create, list and update in other modules) do not timeout. 

Has anyone seen this error before from a serenity save request or know how to fix/troubleshoot it?

**Comment by LSNicholls** [comment]
Does this particular Update request result in some custom code that you need to check?  Whether in Serenity code or in a trigger or something?  I say this because your other updates appear to be okay.  

Also, if you can monitor the number of connections that your app is making to the database, maybe something is wrong there.  I had a "connection leak" that caused this in one app, definitely my own fault, of course! :-)
